.github/workflows/codeql-analysis.yml

```diff
@@ -47,6 +47,22 @@ jobs:
     - name: Building Mystikos
       run: |
         make MYST_PRODUCT_BUILD=1 -j4
-              
+
     - name: Perform CodeQL Analysis
       uses: github/codeql-action/analyze@v1
+      with:
+        upload: false
+        output: sarif-results
+
+    - name: Remove third_party issues from SARIF
+      uses: zbazztian/filter-sarif@master
```

Comment:
Hey Zijie, dont know how frequently this repo would be updated but in the past we have had issues with pinning a reference to the main branch of an outside repo. Should we consider pinning it to a release/commit hash instead?